app/views/root/_location_form.html.erb

```diff
@@ -1,4 +1,7 @@
-<div class="find-nearest">
+<div class="find-nearest"
+      data-module="track-submit"
+      data-track-category="placeSearch"
+      data-track-action="postcodeSearchStarted">
```

Comment:
Category and action are what the analytics guys are searching for specifically. Making them hyphenated here would make the inconsistent with other events in GA.